### ISSUE TITLE
Add alt attribute with "Avatar" as default

### ISSUE THREAD

**Issue by ndarilek** [bengott/meteor-avatar#44]
I'm adding a bunch of accessibility fixes to Telescope, which appears to use this avatars package. Unfortunately, since no alt attribute is specified on avatar images, and no option is available to specify them, they are read by screen readers as strings of random characters.

I'd like to ask that an alt= option be added to the helper. I'd also request that "Avatar" be the default alt value if none is specified, since there should always be an accessible fallback regardless of whether the user configures one. Given that Telescope uses avatars heavily, adding this fallback will go a long way to making its interface more screen reader friendly.

Thanks!

**Comment by bengott** [comment]
Good suggestion, thanks. Please submit a PR if you have time. Otherwise, I'll try to add it myself sometime soon.

**Comment by bengott** [comment]
fixed in version 0.7.5
